### PR TITLE
update Dockerfile for openshift-ci testing

### PR DESCRIPTION
**Pull request by anandrkskd**
**What type of PR is this?**

/kind enhancement

**What does this PR do / why we need it**:
Updating Dockerfile with pre-requisites for 1.7 kuttl tests

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: openshift-ci/build-root/Dockerfile [hunk 1]
```diff
@@ -2,4 +2,27 @@
 
 FROM registry.ci.openshift.org/openshift/release:golang-1.18
 
-RUN yum -y install dnf httpd-tools
+ARG KUBECTL_KUTTL_VERSION=0.12.1
+ARG OPERATOR_SDK_VERSION=1.22.2
```
Comment (varshab1210) [inline openshift-ci/build-root/Dockerfile:6]: Can you update the version to v1.11.0?
https://github.com/redhat-developer/gitops-operator/pull/526/files

Comment (anandrkskd) [reply openshift-ci/build-root/Dockerfile:6]: Operator SDK version?

**Pull request merged**